src/commands/resetlead/ResetLeaderAdd.ts

```diff
@@ -1,55 +1,56 @@
-import * as Util from "../../Util";
-import * as L from "../../Locale";
-import * as discord from "discord.js";
 import * as akairo from "discord-akairo";
+import * as discord from "discord.js";
 import { BotgartCommand } from "../../BotgartCommand";
+import * as L from "../../Locale";
+import * as Util from "../../Util";
 import { WvWMap } from "./ResetRoster";
+import * as ResetUtil from "./ResetUtil";
 
 /**
-Testcases:
+ Testcases:
 
-*/
+ */
 export class AddResetLeader extends BotgartCommand {
-     constructor() {
+    constructor() {
         super("addresetlead", {
-            aliases: ["addresetlead"],
-            quoted: true,
-            args: [
-                {
-                    id: "player",
-                    type: akairo.Argument.union("member", "string")
-                },
-                {
-                    id: "map",
-                    type: (message: discord.Message, phrase: string) => WvWMap.getAllMapNames().includes(phrase) ? phrase : null
-                },
-                {
-                    id: "weekNumber",
-                    type: "integer",
-                    default: -1
-                },
-                {
-                    id: "year",
-                    type: "integer",
-                    default: new Date().getFullYear()
-                }
-            ]
-        }
+                aliases: ["addresetlead"],
+                quoted: true,
+                args: [
+                    {
+                        id: "player",
+                        type: akairo.Argument.union("member", "string")
+                    },
+                    {
+                        id: "map",
+                        type: (message: discord.Message, phrase: string) => WvWMap.getAllMapNames().includes(phrase) ? phrase : null
+                    },
+                    {
+                        id: "weekNumber",
+                        type: "integer",
+                        default: -1
+                    },
+                    {
+                        id: "year",
+                        type: "integer",
+                        default: new Date().getFullYear()
```

Comment:
Probably not an issue, but is the standard `Date` class compatible with your efforts to use `moment` or could it cause problems on edge cases, like on New Years Eve on a leap year, or something like that?

Reply:
I don't think so, but I will change that to use moment and the configured timezone.

Reply:
changed moment (utc) now